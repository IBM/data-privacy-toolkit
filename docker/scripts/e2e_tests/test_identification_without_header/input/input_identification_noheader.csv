foo@gmail.com,John,Paris,Europe,P.O. BOX 1234,A01AA01,Ireland,Male,Married,Bay County,Daughter,12345,127.0.0.1,White,Avenue,http://www.google.com,1985,00647
