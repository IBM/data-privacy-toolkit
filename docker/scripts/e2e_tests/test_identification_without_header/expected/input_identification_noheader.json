{"rawResults":{"Column 15":[{"typeName":"URL","count":1}],"Column 16":[{"typeName":"NUMERIC","count":1}],"Column 13":[{"typeName":"RACE","count":1}],"Column 14":[{"typeName":"UNKNOWN","count":1}],"Column 11":[{"typeName":"NUMERIC","count":1}],"Column 12":[{"typeName":"IP_ADDRESS","count":1}],"Column 10":[{"typeName":"DEPENDENT","count":1}],"Column 17":[{"typeName":"NUMERIC","count":1},{"typeName":"ZIPCODE","count":1}],"Column 1":[{"typeName":"NAME","count":1}],"Column 0":[{"typeName":"EMAIL","count":1}],"Column 3":[{"typeName":"CONTINENT","count":1}],"Column 2":[{"typeName":"CITY","count":1}],"Column 5":[{"typeName":"ATC","count":1}],"Column 4":[{"typeName":"ADDRESS","count":1}],"Column 7":[{"typeName":"GENDER","count":1}],"Column 6":[{"typeName":"COUNTRY","count":1}],"Column 9":[{"typeName":"UNKNOWN","count":1}],"Column 8":[{"typeName":"MARITAL_STATUS","count":1}]},"bestTypes":{"Column 15":{"typeName":"URL","count":1},"Column 16":{"typeName":"NUMERIC","count":1},"Column 13":{"typeName":"RACE","count":1},"Column 14":{"typeName":"UNKNOWN","count":-1},"Column 11":{"typeName":"NUMERIC","count":1},"Column 12":{"typeName":"IP_ADDRESS","count":1},"Column 10":{"typeName":"DEPENDENT","count":1},"Column 17":{"typeName":"NUMERIC","count":1},"Column 1":{"typeName":"NAME","count":1},"Column 0":{"typeName":"EMAIL","count":1},"Column 3":{"typeName":"CONTINENT","count":1},"Column 2":{"typeName":"CITY","count":1},"Column 5":{"typeName":"ATC","count":1},"Column 4":{"typeName":"ADDRESS","count":1},"Column 7":{"typeName":"GENDER","count":1},"Column 6":{"typeName":"COUNTRY","count":1},"Column 9":{"typeName":"UNKNOWN","count":-1},"Column 8":{"typeName":"MARITAL_STATUS","count":1}},"recordCount":1}